test/integration/targets/ecs_certificate/tasks/main.yml

```diff
@@ -0,0 +1,228 @@
+---
+## Verify that integration_config was specified
+- block:
+  - name: Test API User is provided.
+    fail:
+      msg: entrust_api_user should be defined in integration_config.yml
+    when: entrust_api_user is not defined
```

Comment:
Instead of having all these blocks, it might be easier to have them all in one `assert:` statement. The error message won't be as detailled, but it will be a lot shorter.